src/pages/VaultsPage/WithdrawModal.tsx

```diff
@@ -37,18 +37,14 @@ const useStyles = makeStyles(theme => ({
 
 interface WithdrawModalProps {
   onClose: any;
-  openwd: boolean;
+  open?: boolean;
   user?: IUser;
 }
 
-export default function WithdrawModal({ openwd, onClose }: WithdrawModalProps) {
+export default function WithdrawModal({ open, onClose }: WithdrawModalProps) {
   const classes = useStyles();
   const history = useHistory();
 
-  const handleClose = () => {
-    onClose(false);
```

Comment:
@Too-Far @thenerdcat this is what i was referring to in [this comment](https://github.com/coordinape/coordinape/pull/361#discussion_r769772892). the modal shouldn't have to know anything about its callback, but the way this was implemented, it needs to pass one argument and it has to be `false`. this allows the modal to pass some other argument and produce unintended behavior, but more importantly, it also limits reusability--we wouldn't be able to reuse it in different contexts where the parent components have different implementations for `onClose`.

Reply:
ahhh ok, I see what you're saying now. Thank you for clarifying!